l5kit/l5kit/simulation/unroll.py

```diff
@@ -128,26 +155,44 @@ def unroll(self, scene_indices: List[int]) -> List[SimulationOutput]:
                     agents_input_dict = default_collate(list(agents_input.values()))
                     agents_input_dict = {k: v.to(self.device) for k, v in agents_input_dict.items()}
                     agents_output_dict = self.model_agents(agents_input_dict)
+
+                    # for update we need everything as numpy
+                    agents_input_dict = {k: v.cpu().numpy() for k, v in agents_input_dict.items()}
+                    agents_output_dict = {k: v.cpu().numpy() for k, v in agents_output_dict.items()}
```

Comment:
We do that very often in l5kit, I wonder if we shouldn't add a `move_to_numpy` or something similar, that can operate on dict values. Similar to the `move_to_gpu` we have.

Reply:
we don't have `move_to_gpu` either I think :(

Reply:
implemented both!